integration-tests/tests/fixtures/integration_test_logs.py

```diff
@@ -0,0 +1,86 @@
+import os
+import shutil
+import subprocess
+
+import pytest
+from tests.utils.config import (
+    IntegrationTestConfig,
+    IntegrationTestLogs,
+)
+
+
+@pytest.fixture(scope="session")
+def hive_24hr(request, integration_test_config: IntegrationTestConfig) -> IntegrationTestLogs:
+    return _download_and_extract_dataset(
+        request=request,
+        integration_test_config=integration_test_config,
+        name="hive-24hr",
+        tarball_url="https://zenodo.org/records/7094921/files/hive-24hr.tar.gz?download=1",
+    )
+
+
+@pytest.fixture(scope="session")
+def elasticsearch(request, integration_test_config: IntegrationTestConfig) -> IntegrationTestLogs:
```

Comment:
Could we drop elasticsearch and spark_event_logs from this PR since they're large and will slow down initial testing? (Also shrinks the PR.) I think ideally we should eventually figure out a way to make the list of datasets to test configurable---that way users can pick and choose.